src/data/markdown/docs/02 javascript api/09 k6-net-grpc/20-Params.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Params"
+---
+
+*Params* is an object used by the gRPC methods that generate RPC requests. *Params* contains request-specific options like e.g. headers that should be inserted into the request.
+
+| Name | Type | Description |
+|------|------|-------------|
+| `Params.headers` | object | Object with key-value pairs representing custom headers the user would like to add to the request. |
+| `Params.tags` | object | Key-value pairs where the keys are names of tags and the values are tag values. Response time metrics generated as a result of the request will have these tags added to them, allowing the user to filter out those results specifically, when looking at results data. |
+| `Params.timeout` | number | Request timeout to use. Default timeout is 60 seconds (`"60s"`). |
```

Comment:
Is this the same timeout we refer to above as having a default value of `60000` ms? We should make this consistent.

Reply:
:+1: will fix it there

Reply:
Resolved by https://github.com/loadimpact/k6-docs/pull/144/commits/090f7c321be03b0688ef1c6f5b3d2add08c5b7e3